course_info/templates/course_info/partials/people.html

```diff
@@ -180,7 +180,7 @@
 
       <h3>People in {{title}}</h3>
 
-      <uib-alert ng-repeat="person in successes" type="success"
+      <uib-alert id="success-alert" ng-repeat="person in successes" type="success"
```

Comment:
This will create multiple alerts, one for each item in `$scope.successes`, so we need to add a number or something else to the id here.  `id="success-alert-{{$index}}"` will probably work, but that `$index` will track the alert's position in `$scope.successes`.  That is to say, if you remove `success-alert-0`, the ones after it will all see their id change.